proposals/csharp-9.0/init.md

```diff
@@ -242,6 +242,27 @@ Restrictions of this feature:
 - All overrides of a property must have `init` if the base had `init`. This rule
 also applies to interface implementation.
 
+### Readonly structs
+
+`init` accessors (both auto-implemented accessors and manually-implemented
+accessors) are permitted on properties of `readonly struct`s, as well as
+`readonly` properties. `init` accessors are not permitted to be marked
+`readonly` themselves.
```

Comment:
Is the last statement true for all `struct` types or only non-readonly `struct`? Think we should clarify that point.